app/controllers/ChoiceController.scala

```diff
@@ -57,6 +65,16 @@ class ChoiceController @Inject()(
           customsCacheService.cache[Choice](eoriCacheId, choiceId, validChoice).map { _ =>
             validChoice.value match {
               case SupplementaryDec | StandardDec =>
+                val createdDateTime = LocalDateTime.now
+                val model = ExportsCacheModel(
+                  authenticatedSessionId,
+                  UUID.randomUUID().toString,
+                  createdDateTime,
+                  createdDateTime,
+                  validChoice.value
+                )
+                exportsCacheService.update(authenticatedSessionId, model)
```

Comment:
Does this return a future? + if it does we aren't using it

Reply:
This is probably not fully implemented i.e. we should probably handle the future and check it was ok / RIght then redirect. I'll make that change now as I agree it is confusing looking at it as it stands.